engine/cdn/storage/redis/redis.go

```diff
@@ -27,8 +27,14 @@ type Redis struct {
 	bufferType storage.CDNBufferType
 }
 
+const driverName = "redis"
```

Comment:
Can be add a string alias type in SDK with all values at the same place ?
```
type CDNStorageUnitType string
const (
CDNUnitRedis CDNStorageUnitType = "redis"
...
)
```

Reply:
humm, do you think it's really important to have a dependency between the storage subpackage and the whole sdk for just one usage ?

Reply:
i like the idea that a storage bring his driver